docs/dev/create_a_tile.md

```diff
@@ -0,0 +1,321 @@
+| Color Profile      | sRGB                                                                                                                                              |
+| Description        | The maximum number of characters allowed is 300.                                                                                                  |
+
+Follow this template to define the `media` object in the `manifest.json` file which includes an image, a video thumbnail, and a video:
+
+{{< code-block lang="json" filename="manifest.json" collapsible="true" >}}
+"media": [
+      {
+        "media_type": "image",
+        "caption": "A Datadog Marketplace Integration OOTB Dashboard",
+        "image_url": "images/integration_name_image_name.png"
+      },
+      {
+        "media_type": "video",
+        "caption": "A Datadog Marketplace Integration Overview Video",
+        "image_url": "images/integration_name_video_thumbnail.png",
+        "vimeo_id": 123456789
+      },
+    ],
+{{< /code-block >}}
+
+For more information, see [Integrations Assets Reference][22].
+
+## Open a pull request
+
+Before you open a pull request, run the following command to catch any problems with your integration:
+
+```
+ddev validate all <INTEGRATION_NAME>
+```
+
+Complete the following steps:
+
+1. Commit all changes to your feature branch.
+2. Push your changes to the remote repository. 
+3. Open a pull request that contains your integration tile's asset files (including images) in the [`marketplace`][18] or [`integrations-extras`][26] repository. 
+
+After you've created your pull request, automatic checks run to verify that your pull request is in good shape and contains all the required content to be updated.
+
+## Review process
+
+Once your pull request passes all checks, reviewers from the `Datadog/agent-integrations`, `Datadog/marketplace-review`, and `Datadog/documentation` teams provide suggestions and feedback on best practices.
+
+Once you have addressed the feedback and re-requested reviews, these reviewers approve your pull request. Contact the Marketplace team if you would like to preview the tile in your sandbox account. This allows you to validate and preview your tile before your tile is live to all customers.
+
+## Troubleshoot errors
+
+Out-of-the-box integrations in the `integrations-extras` repository can run into validation errors when the forked repository is out of date with the origin. 
+
+To resolve validation errors, update the forked repository on the GitHub web app: 
+
+1. In [GitHub][29], navigate to your forked `integrations-extras` repository.
+1. Click **Sync fork** under the green **Code** dropdown menu and click **Update branch**.
+
+To rebase and push changes:
+
+1. Run `git checkout <your working branch>`.
+1. Run `git rebase master`.
+1. If there are any merge conflicts, resolve them. Then, run `git push origin <working branch> -f`.
```

Comment:
~~~suggestion
1. Update your local master branch:
   ```shell
   git checkout master
   git pull origin master
   ```
1. Merge `master` into your feature branch:
   ```shell
   git checkout <your working branch>
   git merge master
   ```
1. If there are any merge conflicts, resolve them. Then, run `git push origin <your working branch>`.
~~~
They should pull down the updated `master` branch before they rebase/merge, or nothing will happen. I don't know if I'd recommend a rebase over a standard master merge. If the user is not an experienced git user, a rebase could lead them into some trouble, especially if a lot of time has passed since they forked. It also avoids a history change and a force push.